adi/ad719x.py

```diff
@@ -89,14 +89,15 @@ def offset(self, value):
     def to_volts(self, index, val):
         """Converts raw value to SI."""
         _scale = self.channel[index].scale
+        _offset = self.channel[index].offset
 
         ret = None
 
         if isinstance(val, np.int32):
-            ret = val * _scale
+            ret = (val + _offset) / 1000 * _scale
```

Comment:
I'm alittle confused on the div by 1000. Standard units for IIO is mV. This will make it uV.